src/applications/personalization/dashboard-2/components/CTALink.jsx

```diff
@@ -13,11 +17,14 @@ const CTALink = ({ ariaLabel, href, text, onClick, newTab }) => {
       onClick={onClick || undefined}
       className="vads-u-margin-top--2 vads-u-display--inline-block"
     >
-      {text}
-      <i
-        aria-hidden="true"
-        className="fas fa-xs fa-chevron-right vads-u-margin-x--1"
-      />
+      {`${firstWords} `}
+      <span style={{ whiteSpace: 'nowrap' }}>
```

Comment:
We don't have a util class for `nowrap`. But this fix works:

**408px wide**
<img width="406" alt="408" src="https://user-images.githubusercontent.com/20728956/113795682-4ea38480-9702-11eb-8392-a70cd3c00e69.png">

**409px wide**
<img width="403" alt="409" src="https://user-images.githubusercontent.com/20728956/113795699-5531fc00-9702-11eb-857e-f744ec87901e.png">

Reply:
You can play with this locally by changing [this test](https://github.com/department-of-veterans-affairs/vets-website/blob/3b60bb1a209a4bb0b1395bfb301ec498ee882b4f/src/applications/personalization/dashboard-2/tests/e2e/loa3.cypress.spec.js#L86) to `.only` and then mess with the viewport,  `cy.viewport(409, 800)`, on line 87